src/tests/electron/tests/android-setup-locate-adb.test.ts

```diff
@@ -45,14 +45,8 @@ describe('Android setup - locate adb', () => {
         await dialog.waitForDialogVisible('prompt-connected-start-testing');
     });
 
-    it.each([true, false])(
-        'should pass accessibility validation with highContrastMode=%s',
-        async highContrastMode => {
-            await app.setHighContrastMode(highContrastMode);
-            await app.waitForHighContrastMode(highContrastMode);
-
-            const violations = await scanForAccessibilityIssues(dialog);
-            expect(violations).toStrictEqual([]);
-        },
-    );
+    it('should pass accessibility validation in both contrast modes', async () => {
+        await scanForAccessibilityIssues(app, true);
```

Comment:
We're still repeating the pattern of scan(app, true) scan(app, false). Would it make sense to move the true/false logic into a helper method like `scanForAccessibilityIssuesInAllModes(app)`, and call that from the tests. If a future change added a new HC mode, none of the tests would need to be updated.